src/main/kotlin/no/nav/familie/ef/iverksett/infotrygd/SendPerioderTilInfotrygdTask.kt

```diff
@@ -29,6 +30,9 @@ class SendPerioderTilInfotrygdTask(private val infotrygdFeedClient: InfotrygdFee
     override fun doTask(task: Task) {
         val iverksett = iverksettingRepository.hent(UUID.fromString(task.payload))
         val stønadstype = iverksett.fagsak.stønadstype
+        if (stønadstype != StønadType.OVERGANGSSTØNAD) {
```

Comment:
Hvorfor?

Reply:
Ja, det burde vært med i commitmeldingen. Periodene brukes for å legges ut på en kafka tror jeg, som barnetrygd sen bruker. 
https://confluence.adeo.no/pages/viewpage.action?pageId=385067073# det finnes kun periode-hendelser for stønadstypen overgangsstønad